docs/docs/modeling.md

```diff
@@ -106,6 +106,44 @@ classDiagram
 
 ## ShareHouse - シェアハウス
 
+The Landlord can register a ShareHouse which they own.
+
+### ShareHouse ID (Identifier)
+
+The ShareHouse ID is the UUID.
+
+### ShareHouse Name
+
+The ShareHouse has a default name when created without specifying one.
+
+Constraints：
+
+- Uniqueness
+- Greater than or equal to 1 character
+- Less than or equal to 15 characters
+- Can be modified by Landlord as they like
+- Uppercase and lowercase letters are recognized as the same characters
+
+### rotationCycle
+
+The rotationCycle must be set to Weekly or Fortnightly.
+
+Constraints：
+
+- Cannot be assigned to a Category until the rotationCycle is set.
```

Comment:
I'm not sure about this part

Reply:
I agree with Yuki.
My understanding was that setting the category comes first, and determining the rotation cycle comes last.